src/Features/ERDDAP/Platform/Observations/WaterLevel/WLErddapObservationTable.tsx

```diff
@@ -72,7 +113,17 @@ export const WLErddapObservationTable: React.FC<Props> = ({
       {waterLevelTimeseries && (
         <TableItem key="WL-ts" timeSeries={waterLevelTimeseries} platform={platform} unitSystem={unitSystem} />
       )}
-
+      {highTide && lowTide && lowTide < highTide ? (
```

Comment:
This ternary is making my head spin. I think you are trying to control if the components display based on if both high and low tide exist, but change the order if low is before high? 

I feel like if tides are not available we will get two list group items with no data.

Reply:
Hah yeah this was not my finest work when it comes to readable code.  I was trying to order them based on which is chronologically coming first. But since we are putting them in a table below the graph, I think this will be obsolete. But I will try to stray (far) away from complex ternaries like this